index.html

```diff
@@ -135,35 +135,56 @@ <h4 class="item-link__title">Open edX Release Notes</h4>
                     </div>
                   </a>
                 </li>
+                
+                <li class="item-link">
+                  <a class="item-link__action" href="http://edx.readthedocs.org/projects/open-edx-building-and-running-a-course/en/named-release-birch/">
+                    <h4 class="item-link__title">Building and Running an Open edX Course: Birch Release</h4>
+                    <div class="item-link__copy">
```

Comment:
should this also have the date?
<div class="item-link__updated-date">Last Updated: February 24, 2015 for the Birch Release</div>